.buildkite/Makefile

```diff
@@ -0,0 +1,5 @@
+# Builds all dhall entrypoints
+check:
+	dhall <<< './src/Prepare.dhall'
```

Comment:
You should handle the stderr of this to identify which file failed (hint: writing a `check-dhall.sh` would be helpful here). At minimum, can you pipe the stdout of this to `/dev/null`?

Reply:
Actually I like the feedback in stderr and stdout during the dev loop (anecdotally) even if it succeeds, it's nice to be able to eyeball all the yaml. Is this okay?

Reply:
You can leave it for now, but I would still like to see the explicit call out of which file failed like I mentioned. Otherwise you don't know what file failed from the output, do you?

Reply:
It short circuits on failure and you see something like this:

<img width="654" alt="Screen Shot 2020-05-05 at 2 11 00 PM" src="https://user-images.githubusercontent.com/515445/81100280-492e3080-8eda-11ea-9c78-d05c0a9624ac.png">

The error messages are very clear 👍